js/mobile-nav.js

```diff
@@ -0,0 +1,43 @@
+namespacer("restaurantWeek");
+
+restaurantWeek.mobileNav = ($ => {
+  let self = {};
+
+  const init = options => {
+    self.options = self.options || {};
+    self.options.mobileNavButtonSelector =
+      self.options.mobileNavButtonSelector || ".mobile-nav-btn";
+    self.options.navigationListSelector =
+      self.options.navigationListSelector || ".navigation-list";
+
+    $(document).on(
+      "click",
+      self.options.mobileNavButtonSelector,
+      onMobileNavClick
+    );
+  };
+
+  const onMobileNavClick = e => {
```

Comment:
Please use a better name for the event than "e". For example, "clickEvent".

Reply:
good call, done